web/locales/en/plugin__netobserv-plugin.json

```diff
@@ -398,6 +399,7 @@
   "Time elapsed between DNS request and response.": "Time elapsed between DNS request and response.",
   "DNS Response Code": "DNS Response Code",
   "DNS RCODE name from response header.": "DNS RCODE name from response header.",
+  "DNS error number returned by bpf_skb_load_bytes function.": "DNS error number returned by bpf_skb_load_bytes function.",
```

Comment:
I would say error code returned from DNS tracker ebpf hook because it doesn't have to be return from `bpf_skb_load_bytes` plus this too much details to show here IMO

Reply:
Sure, sounds better ! https://github.com/netobserv/network-observability-console-plugin/pull/425/commits/8ffc6cbff7ed87fbba9cb3992fe0fdd26fa3a1c0